vslib/src/SwitchStateBase.cpp

```diff
@@ -1820,9 +1820,11 @@ sai_status_t SwitchStateBase::refresh_read_only(
         switch (meta->attrid)
         {
             case SAI_SYSTEM_PORT_ATTR_TYPE:
-            case SAI_SYSTEM_PORT_ATTR_CONFIG_INFO:
             case SAI_SYSTEM_PORT_ATTR_PORT:
                 return SAI_STATUS_SUCCESS;
+
+            case SAI_SYSTEM_PORT_ATTR_CONFIG_INFO:
+                return refresh_system_port_config_info(meta, object_id);
```

Comment:
SAI_SYSTEM_PORT_ATTR_CONFIG_INFO is create_only attribute, it should not be refreshed

Reply:
Yes. I'll remove this

Reply:
Fixed